qiskit/algorithms/minimum_eigensolvers/__init__.py

```diff
@@ -0,0 +1,48 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+=====================================================================
+Minimum Eigensolvers Package (:mod:`qiskit.algorithms.minimum_eigensolvers`)
+=====================================================================
+
+.. currentmodule:: qiskit.algorithms.minimum_eigensolvers
+
+Minimum Eigensolvers
+================
+.. autosummary::
+   :toctree: ../stubs/
+
+   MinimumEigensolver
+   NumPyMinimumEigensolver
+   QAOA
+   VQE
+   
+.. autosummary::
+   :toctree: ../stubs/
+
+   MinimumEigensolverResult
+   NumPyMinimumEigensolverResult
+   VQEResult
+
```

Comment:
Forgot to make it into a docstring in my previous suggestion. This fixes the lint error:

```suggestion
"""
============================================================================
Minimum Eigensolvers Package (:mod:`qiskit.algorithms.minimum_eigensolvers`)
============================================================================

.. currentmodule:: qiskit.algorithms.minimum_eigensolvers

Minimum Eigensolvers
====================

.. autosummary::
   :toctree: ../stubs/

   MinimumEigensolver
   NumPyMinimumEigensolver
   QAOA
   VQE

Results
=======

 .. autosummary::
    :toctree: ../stubs/

   MinimumEigensolverResult
   NumPyMinimumEigensolverResult
   VQEResult
"""

```

Reply:
Sorry only saw your suggestions after already pushing fixes addressing this.